std/datetime.d

```diff
@@ -6375,6 +6386,14 @@ public:
         //static assert(__traits(compiles, ist + duration));
         static assert(__traits(compiles, cst - duration));
         //static assert(__traits(compiles, ist - duration));
+
+				// Test DST
+				auto beforeDST = SysTime(DateTime(2015, 11, 1), TimeZone.getTimeZone("America/Los_Angeles")); 
+				assert((beforeDST + dur!"hours"(1)).toSimpleString == "2015-Nov-01 01:00:00-07:00");
+				assert((beforeDST + dur!"hours"(2)).toSimpleString == "2015-Nov-01 01:00:00-08:00");
+				auto duringDST = SysTime(DateTime(2015, 11, 1, 1), TimeZone.getTimeZone("America/Los_Angeles")); 
+				assert((duringDST + dur!"hours"(1)).toSimpleString == "2015-Nov-01 01:00:00-08:00");
+				assert((duringDST + dur!"hours"(2)).toSimpleString == "2015-Nov-01 02:00:00-08:00");
```

Comment:
`toSimpleString` has nothing to do with `opBinary`, so these tests are inappropriate. The behavior with regards to DST is tested quite thoroughly in the unit tests relating to the `TimeZone`s. These tests are redundant.